public/app/features/dashboard/components/PanelEditor/usePanelLatestData.ts

```diff
@@ -35,6 +35,7 @@ export const usePanelLatestData = (panel: PanelModel, options: GetDataOptions):
      * Adding separate options to dependencies array to avoid additional hook for comparing previous options with current.
      * Otherwise, passing different references to the same object may cause troubles.
      */
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [panel, options.withFieldConfig, options.withTransforms]);
```

Comment:
Nit: should we have a dependency on the `setLatestData` here?

Reply:
and maybe destructuring the options object.

Reply:
> Nit: should we have a dependency on the `setLatestData` here?

Nah, the setter from useState is always same so it doesn't need to be added to the dependency array. See https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often

> and maybe destructuring the options object.
Not sure about that. @dprokop do you have any thoughts about it?